packages/flutter/lib/src/material/icon_button.dart

```diff
@@ -197,6 +197,8 @@ class IconButton extends StatelessWidget {
     this.splashColor,
     this.disabledColor,
     required this.onPressed,
+    this.onHover,
+    this.onLongPress,
```

Comment:
These properties are added multiple IconButton variants so we should make sure all variants. If by mistake one of the contributors is modified tests would cover the mistake.

Reply:
I added it to all variants

Reply:
Multiple variants in the Column  in the same tests.

Reply:
https://github.com/flutter/flutter/pull/160032#pullrequestreview-2502469365